bin/deploy-vhost

```diff
@@ -323,6 +348,26 @@ END
 END
             }
     }
+
+    # Add all the parameters as structured data
+
+    push $conf->{struct_database_configs}{internal}, (
```

Comment:
This won't work, as you're trying to push an array on to a scalar, you want to push an arrayref onto an array.